services/provider/server/server.go

```diff
@@ -1236,6 +1236,20 @@ func (s *OCSProviderServer) getDesiredCephConnection(
 		},
 		Spec: csiopv1a1.CephConnectionSpec{
 			Monitors: monIps,
+			ReadAffinity: &csiopv1a1.ReadAffinitySpec{
```

Comment:
@OdedViner having this here makes every connected client have readaffinity but only a corresponding storageclass will use this and no side effect for other SCs?

Reply:
@leelavg readaffinity will be enabled for all the storageclasses.